Source/Scene/FXAA.js

```diff
@@ -54,6 +54,9 @@ define([
         this._viewport = new BoundingRectangle();
         this._rs = undefined;
 
+        this._useScissorTest = undefined;
```

Comment:
Throughout, `_useScissorTest` is a boolean, right?  Initialize it as such, e.g., `= false`, unless this creates an issue.